apps/challenges/aws_utils.py

```diff
@@ -956,12 +958,19 @@ def create_eks_cluster(challenge):
             config_text = yaml.dump(cluster_config, default_flow_style=False)
             config_file = NamedTemporaryFile(delete=True)
             config_file.write(config_text.encode())
-            ChallengeEvaluationCluster.objects.create(
-                challenge=challenge_obj,
-                name=cluster_name,
-                cluster_endpoint=cluster_ep,
-                cluster_ssl=cluster_cert,
-            )
+            try:
+                ChallengeEvaluationCluster.objects.get(
```

Comment:
Why are we executing this try block?

Reply:
@RishabhJain2018  The use case is to create the fields in the model when they don't already exist. And we are not updating anything in the try block because, if a challenge name is changed, the Kubernetes cluster won't be renamed.